index.html

```diff
@@ -53,23 +56,27 @@
     </div>
     <div class="collapse navbar-collapse">
       <ul class="nav navbar-nav">
+        <!--
         <li>
           <a href="https://github.com/gitbucket/gitbucket"><span class="mega-octicon octicon-mark-github octicon-btn"></span>GitHub</a>
         </li>
-        <li><a href="https://gitter.im/gitbucket/gitbucket"><span class="mega-octicon octicon-comment octicon-btn"></span>Chat</a></li>
-        <li><a href="https://gitbucket.herokuapp.com/"><span class="mega-octicon octicon-rocket octicon-btn"></span>Demo</a></li>
-        <li><a href="https://gitbucket.github.io/gitbucket-news/"><span class="mega-octicon octicon-rss octicon-btn"></span>News</a></li>
-        <li><a href="http://gitbucket-plugins.github.io/"><span class="mega-octicon octicon-plug octicon-btn"></span>Plugins</a></li>
+        -->
+        <li>&nbsp;&nbsp;&nbsp;</li>
+        <li><a href="https://gitter.im/gitbucket/gitbucket" title="Get GitBucket support on Chat!"><span class="mega-octicon octicon-comment octicon-btn"></span>Chat</a></li>
+        <li><a href="https://gitbucket.herokuapp.com/" title="Try a live Demo of GitBucket!"><span class="mega-octicon octicon-rocket octicon-btn"></span>Demo</a></li>
+        <li><a href="https://gitbucket.github.io/gitbucket-news/" title="Read the latest GitBucket News!"><span class="mega-octicon octicon-rss octicon-btn"></span>News</a></li>
+        <li><a href="http://gitbucket-plugins.github.io/" title="The GitBucket Plug-in Repository!"><span class="mega-octicon octicon-plug octicon-btn"></span>Plug-ins</a></li>
+        <li><a href="https://github.com/gitbucket/gitbucket/wiki" title="GitBucket Documentation!"><span class="mega-octicon octicon-book octicon-btn"></span>Documentation</a></li>
       </ul>
-      <!--
+
       <div class="navbar-right">
-        <a class="lg-block navbar-brand btn btn-skelton margin-top minus-margin-right"
-           href="https://github.com/gitbucket/gitbucket" target="_blank">
+        <a class="lg-block navbar-brand btn btn-skeleton margin-top minus-margin-right github"
+           href="https://github.com/gitbucket/gitbucket" target="_blank" title="Development on GitHub">
           <span class="octicon octicon-mark-github octicon-btn"></span>
-          Fork me on GitHub
+          GitHub
         </a>
       </div>
```

Comment:
It has indeed, but it's the right position to put a link to GitHub.
IMHO It's much better there on the right side, separated and far away from GitBucket logo.